pkg/catalog/routes.go

```diff
@@ -104,6 +104,15 @@ func (mc *MeshCatalog) ListAllowedOutboundServices(sourceService service.MeshSer
 
 // ListAllowedOutboundServicesForIdentity list the services the given service account is allowed to initiate outbound connections to
 func (mc *MeshCatalog) ListAllowedOutboundServicesForIdentity(identity service.K8sServiceAccount) []service.MeshService {
+	allowedServices := []service.MeshService{}
+
+	if mc.configurator.IsPermissiveTrafficPolicyMode() {
+		for _, svc := range mc.kubeController.ListServices() {
+			allowedServices = append(allowedServices, utils.K8sSvcToMeshSvc(svc))
+		}
```

Comment:
nit: I think we can have a method `ListMeshServices()` that handles the logic in these lines and returns `[]service.MeshService`. I see more use case in the future where this method might be useful